certwebapp/settings.py

```diff
@@ -55,7 +54,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [os.path.join(BASE_DIR, 'templates', '../accounts/templates')],
```

Comment:
leave it empty:
DIR:[ ]

Reply:
Ξέρει που ειναι τα templates και τα static, επειδη το βάλαμε το accounts sto INSTALLED_APPS.